Tests/Marketplace/common.py

```diff
@@ -163,3 +164,104 @@ def wait_until_not_updating(client: demisto_client,
                 return False
     logging.info(f"Exiting after exhausting the allowed time:{maximum_time_to_wait} seconds")
     return False
+
+
+def send_api_request_with_retries(
```

Comment:
have you considered somehow avoiding the code-dup between this one and `generic_request_with_retries()`?

Reply:
I tried but it will do the code more ugly. 
I opened a ticket that may be helpfull: https://jira-hq.paloaltonetworks.local/browse/CIAC-8674